blogs/scripted-windows/install-windows-configmap.yaml

```diff
@@ -67,7 +67,7 @@ data:
                                 devices:
                                   disks:
                                     - disk:
-                                        bus: virtio
+                                        bus: sata
```

Comment:
we want the disk to work with virtue drivers if the order of the disk is correct this should work cause windows will load the drivers from the guest utils

this is happening during the windows setup and in the XML

https://github.com/RHsyseng/cnv-integration/blob/main/blogs/windows-unattended/autounattend.xml#L71